addons/mail/static/src/discuss/core/common/discuss_notification_settings.js

```diff
@@ -0,0 +1,33 @@
+import { Component, useState } from "@odoo/owl";
+import { useService } from "@web/core/utils/hooks";
+
+export class DiscussNotificationSettings extends Component {
+    static props = {};
+    static template = "mail.DiscussNotificationSettings";
+
+    setup() {
+        this.store = useState(useService("mail.store"));
+    }
+
+    onChangeDisplayMuteDetails() {
+        // set the default mute duration to forever when opens the mute details
+        if (!this.store.settings.mute_until_dt) {
+            this.store.settings.setMuteDuration(
+                this.store.settings.MUTES.find((m) => m.id === "forever").value
+            );
```

Comment:
Ideally "Until I turn it back on" should be selected in the list in this case rather than "select duration".